node/core/backing/src/lib.rs

```diff
@@ -16,6 +16,8 @@
 
 //! Implements a `CandidateBackingSubsystem`.
 
+#![deny(unused_extern_crates, unused_results)]
```

Comment:
unused_extern_crates are only for unused `extern crate foo` declarations, it doesn't have an effect on unused `Cargo.toml` dependencies and is not helpful for the 2018 edition IIUC.

There is `unused_crate_dependencies` https://github.com/rust-lang/rust/pull/72342, but it has false positives https://github.com/rust-lang/rust/issues/57274#issuecomment-624839355.

Reply:
Let me check how that goes, if there are no false positives, I'd say go for it, otherwise I'll drop it for now.